tests/e2e/validator.go

```diff
@@ -167,6 +174,49 @@ func (v *validator) createKeyFromMnemonic(name, mnemonic string) error {
 	return nil
 }
 
+func (c *chain) addAccountFromMnemonic(counts int) error {
+	val0ConfigDir := c.validators[0].configDir()
+	kb, err := keyring.New(keyringAppName, keyring.BackendTest, val0ConfigDir, nil, cdc)
+	if err != nil {
+		return err
+	}
+
+	keyringAlgos, _ := kb.SupportedAlgorithms()
+	algo, err := keyring.NewSigningAlgoFromString(string(hd.Secp256k1Type), keyringAlgos)
+	if err != nil {
+		return err
+	}
+
+	for i := 0; i < counts; i++ {
+		name := fmt.Sprintf("acct-%d", i)
```

Comment:
To avoid confusion I might not just name each one by number...for example the wallet relayer account should be separate than alternative accounts we might want to use just for testing purposes...lmk what you think